index.js

```diff
@@ -7,16 +7,25 @@ module.exports = function (emitter, type) {
   var emit = emitter && emitter.emit
   if (typeof emit !== 'function') return
 
-  emitter.emit = function (event) {
+  emitter.emit = function (event, ...args) {
     var end = Date.now()
     var diff = start === null ? 0 : end - start
     start = end
 
-    console.error(
-      chalk.yellow((type || emitter.constructor.name) + ':'),
-      chalk.white(event),
-      chalk.red('+' + diff + 'ms')
-    )
+    if (args.length > 0) {
+      console.error(
+        chalk.yellow((type || emitter.constructor.name) + ':'),
+        chalk.white(event),
+        chalk.white('(' + args.join(', ') + ')'),
```

Comment:
I think it looks better if we make this grey so it doesn't look like the event name just next to it 😃 

```suggestion
        chalk.gray('(' + args.join(', ') + ')'),
```

Reply:
Changed :)